src/structs/shared_structs.go

```diff
@@ -22,6 +22,15 @@ type Game struct {
 	Word     string       `json:"word" bson:"word"`
 }
 
+type SharedGame struct {
+	SharedGameID string            `json:"sharedGameID" bson:"sharedgameid"`
+	HostID       string            `json:"hostID" bson:"hostid"`
+	Games        map[string]string `json:"Games" bson:"Games"`
+	State        string            `json:"state" bson:"state"`
+	WinnerID     string            `json:"winnerID" bson:"winnerid"`
+	Word         string            `json:"word" bson:"word"`
```

Comment:
For consistency:
```suggestion
	SharedGameId string            `json:"sharedGameId" bson:"sharedgameid"`
	HostId       string            `json:"hostId" bson:"hostid"`
	Games        map[string]string `json:"games" bson:"games"`
	State        string            `json:"state" bson:"state"`
	WinnerId     string            `json:"winnerId" bson:"winnerid"`
	Word         string            `json:"word" bson:"word"`
```

Reply:
Ignore the previous comment, we should do what we talked about (ID for the variable names and JSON, id for the BSON)